src/main/webapp/help/help-maxLines.html

```diff
@@ -1,5 +1,5 @@
 <div>
   <p>The maximum number of log lines to send to Logstash.<br/>
   If the log is bigger than this, only the most recent lines are sent.<br/>
-  -1 indicates there is no maximum (for very large logs, consider using the Logstash Job Property instead).</p>
+  -1 indicates there is no maximum (for very large logs, consider using the Logstash Job Property instead or the logstash step).</p>
```

Comment:
I'm confused: I though the property is only used internally. Can users setup the property themselves directly?

Reply:
The JobProperty has a small UI part that you can check in a freestyle job. It currently has no other things to configure.

Reply:
Maybe because we had that migration away from the BuildWrapper. This has no longer any UI.

Reply:
> This has no longer any UI.

Shouldn't we remove the reference to the Job Property then, or do you mean one could still add it via script?

Reply:
The JobProperty is still there and you can check it in your job. But it has no other configuration currently. The BuildWrapper class still exists for backwards compatibility but has no UI at all. You can't configure it any longer. Afaik all references in the UI to the Buildwrapper are gone.